components/form-builder/panel/ElementPanel.tsx

```diff
@@ -438,7 +442,7 @@ export const ElementWrapper = ({ item }: { item: ElementTypeWithIndex }) => {
           </ModalButton>
         )}
       >
-        {modals[item.index] && (
+        {!isRichText && modals[item.index] && (
           <ModalForm
             item={item}
             properties={modals[item.index]}
```

Comment:
I'm not sure if it's here or elsewhere, but changing a question field to `richText` causes a console error.

<img width="870" alt="Screen Shot 2022-09-21 at 13 21 34" src="https://user-images.githubusercontent.com/2454380/191571150-d40c5aad-b37b-4626-a965-53daa2cefd1b.png">

Reply:
🤔 --- I don't see a way to add an optional prop type --- some online says to use `any`  

https://github.com/cds-snc/platform-forms-client/pull/1056/commits/992047a4bc6ea2fb8df2c7788aa6d8ce19d6421e

Reply:
In this case we don't need the modal so no children